components/button.json

```diff
@@ -105,25 +105,25 @@
       },
       "#hovered":{
         "background":"@theme-button-cancel-hover",
-        "text": "@theme-text-primary-normal"
+        "text": "@theme-text-button-normal"
```

Comment:
Shouldn't this change be in cancel-fill, not cancel-outline?

Reply:
<img width="627" alt="Screenshot 2021-07-23 at 14 32 15" src="https://user-images.githubusercontent.com/87858909/126789247-e298f3da-e72e-4308-9a4f-564a2b47e432.png">

its white (theme-text-button-normal) in both  hover and pressed

Reply:
Ah sorry, yes, in cancel-fill _as well as_ cancel-outline